tests/unit/test_transfer_client.py

```diff
@@ -29,6 +59,7 @@ def setUp(self):
         Creates a list for tracking cleanup of assets created during testing
         Sets up a test endpoint
         """
+        super(BaseTransferClientTests, self).setUp()
```

Comment:
Good catch! We're not really relying on the IO capturing for much of anything, but this way we won't see warnings or other noise.